.travis.yml

```diff
@@ -95,14 +97,21 @@ jobs:
     - stage: release-dev
       script: bin/s3 sync s3://build-artifacts build/package
       name: "Update for build-artifacts"
+      before_deploy:
+        - git tag -f dev
```

Comment:
Dont You want to add this to script?

Reply:
I think that it's more readable to keep it here. 
It's not too big to have an extra script for that, and it never used for the second time somewhere else.